packages/@o3r/schematics/src/utility/loaders.ts

```diff
@@ -6,6 +7,34 @@ import * as path from 'node:path';
 import type { PackageJson } from 'type-fest';
 import type { WorkspaceProject, WorkspaceSchema } from '../interfaces/index';
 
+function findFilenameInTreeRec(memory: Set<FileEntry>, directory: DirEntry, fileMatchesCriteria: (file: string) => boolean, ignoreDirectories: string[]) {
+  if (ignoreDirectories.some(i => directory.path.includes(i))) {
```

Comment:
`includes` or `startsWith` ?

Reply:
I prefer includes so that we apply it to all levels (ex node_modules inside app folders)

Reply:
findFilenameInTreeRec(..., ['directory-to-ignore'])
actual code will ignore 'path/not-directory-to-ignore'

```suggestion
  if (ignoreDirectories.some(i => directory.path.split(path.posix.sep).includes(i))) {
```

Reply:
you are right